bridge/client/app/project-board/project-board.component.html

```diff
@@ -114,7 +174,7 @@ <h2 class="m-0" [textContent]="service.serviceName"></h2>
                   </div>
                 </div>
               </dt-info-group-title>
-              <p class="m-0 mb-1 mt-1" *ngIf="getLatestDeployment(project, service) as deployment">
+              <p class="m-0 mb-1 mt-1" *ngIf="project.getLatestDeployment(service, project.stages[0]) as deployment; else noDeployment">
```

Comment:
Maybe a keptn-specific - do you, at this point, always know for sure, that there is at least one stage? Otherwise  you could get a index out of range issue here

Reply:
There is always at least one stage, because you have to specify the stages already when creating the project (even before onboarding a service), see https://keptn.sh/docs/0.6.0/manage/project/#create-a-project